modules/ldap-syncing-augmented-activedir.adoc

```diff
@@ -0,0 +1,262 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-syncing-augmented-activedir_{context}"]
+= Syncing groups using the Active Directory schema
```

Comment:
This has the same title as the module above it. I assume "augmented" needs to be added in here.